### PR TITLE
gpad parser now takes taxon element out of the list instead of passing on the entire list

### PR DESCRIPTION
**Pull request by dougli1sqrd**
Fixes issue #489

### CODE REVIEW

**Comment by kltm** [comment]
Looks good to me...but I'm new here. Will wait on tests.

File: setup.py [hunk 1]
```diff
@@ -54,7 +54,7 @@
         'marshmallow==3.0.0b11',
         'scipy',
         'pandas==1.0.5',
```
Comment (deepakunni3) [inline setup.py:56]: Can we also remove the version lock for Pandas? This has been a problem before when other tools use Ontobio as a dependency.

Comment (deepakunni3) [reply setup.py:56]: And the lock on networkx too :)

**Review by deepakunni3 — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (dougli1sqrd) [reply setup.py:56]: oh sure

**Review by deepakunni3 — APPROVED** [review]

**Pull request merged**